src/plugins/intel_gpu/src/plugin/device_config.cpp

```diff
@@ -69,6 +69,12 @@ void Config::UpdateFromMap(const std::map<std::string, std::string>& configMap)
         const auto hints = perfHintsConfig.SupportedKeys();
         if (hints.end() != std::find(hints.begin(), hints.end(), key)) {
             perfHintsConfig.SetConfig(key, val);
+        } else if (key == ov::hint::inference_precision) {
+            std::stringstream ss(val);
+            ss >> inference_precision;
+            OPENVINO_ASSERT(inference_precision == ov::element::f16 ||
+                            inference_precision == ov::element::f32,
```

Comment:
Isn't this needed for i8/u8?

Reply:
No. This option allows conversion for FP precisions only and doesn't change execution precision of quantized primitives.